### PR TITLE
fix: remove code forcing selected heading on go-around phase

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* Experimental fix for AP not following nav line

Discord username (if different from GitHub): someperson#4953

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 1]
```diff
@@ -1679,6 +1697,7 @@ class FMCMainDisplay extends BaseAirliners {
         return this._routeReservedPercent;
     }
     trySetRouteReservedPercent(s) {
+        upda;
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js]: what the heck is that?

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js [hunk 2]
```diff
@@ -1183,13 +1201,13 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
...
-                        SimVar.SetSimVarValue("L:A32NX_GOAROUND_NAV_OVERRIDE", "bool", 1);
-                        this._onModeSelectedHeading();
-                    }
+                    // speed = this.getPerfGreenDotSpeed();
+                    // //delete override logic when we have valid nav data -aka goaround path- after goaround!
+                    // if (SimVar.GetSimVarValue("L:A32NX_GOAROUND_NAV_OVERRIDE", "bool") === 0) {
+                    //     console.log("only once per goaround override to HDG selected");
+                    //     SimVar.SetSimVarValue("L:A32NX_GOAROUND_NAV_OVERRIDE", "bool", 1);
+                    //     this._onModeSelectedHeading();
+                    // }
```
Comment (MisterChocker) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js:1210]: This is needed for the FMAs to function properly.

**Comment by ImenesFBW** [comment]
This unfortunately doesn't fix the AP problem for me.

**Comment by z0rgmeister** [comment]
Category: QA Tester
Name: z0rgmeister#8972
Date of testing: Current date 29/11/2020
Version of the sim: 1.11.6.0
PR Tested: #2122
Object of test: AP ability to follow NAV path

Testing technique: Full LPFR-LPPT flight from C&D
Steps you've made to test the Pull Request:
1. Instant INS Startup. MCDU programming, including all the pages and manual flight plan input.
2. Took off with FD1+2, manual to Accel ALT. AP1 on. Observed horizontal flight path following. (Climb rate in excess of 4500fpm too)
3. Followed  FP until TOD, ordered DIR to IAF (skipping 2 or 3 WPs along the way). Observed horizontal flight path OK. AP2 on for argument's sake, no change.
4. Changed approach runway, forcing an extra 40 NM on the flight path. Observed AP following horizontal flight path OK. Back to AP1.
5. Persistent approach issues (trying to go direct to runway, overshooting centrelines and difficulty in following horizontal instructions verified present)

Flight notes (If required): All OK except approach as per above, but that is outside the scope of the PR and it is an historical issue.

Issues (If Any): Approach as per above. Slight "wobble" on level CRZ, possibly a regression to previous states.
Overall Rating: Good
Conclusions of the testing: Was able to do a full flight from C&D, around 160NM in total. AP on after Throttle reduction altitude, able to follow the flight path too. Slight wobble (dutch roll issue?) during crz phase.
DIR works as I tested it on purpose. Was even able to change the selected approach without it having too many issues. It generally follows the green line. Generally...which is more than what we can say for the default aircraft.
It's when approaching final that all the old issues of flying straight to the runway, overshooting lines and sometimes wanting to turn too early in LOC pop up. But overall a significant improvement to the post patch state of affairs.

**Comment by Benjozork** [comment]
Not the issue after all :/

**Pull request closed without merging**